python/dgl/graphbolt/impl/ondisk_dataset.py

```diff
@@ -528,23 +528,34 @@ class BuiltinDataset(OnDiskDataset):
         The root directory of the dataset. Default ot ``datasets``.
     """
 
+    # For dataset that is smaller than 30GB, we use the base url.
+    # Otherwise, we use the accelerated url.
     _base_url = "https://data.dgl.ai/dataset/graphbolt/"
+    _accelerated_url = (
+        "https://dgl-data.s3-accelerate.amazonaws.com/dataset/graphbolt/"
+    )
     _datasets = [
```

Comment:
I would rename this to _small_datasets

Reply:
and `_datasets = _small_datasets + _large_datasets`

Reply:
actually ,it's small+medium vs large

Reply:
either way works, just a nit :)